src/App.tsx

```diff
@@ -1,223 +1,226 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React,
+{
+  useContext,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
 import { AuthContext } from './components/Auth/AuthContext';
+import {
+  getTodos,
+  addTodo,
+  deleteTodo,
+  updateTodo,
+} from './api/todos';
+
+import { Todo } from './types/Todo';
+import { FilterType } from './types/FilterType';
+
+import { Todolist } from './components/TodoList';
+import { Filter } from './components/Filter';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Header } from './components/Header';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState<FilterType>(FilterType.ALL);
+  const [hasError, setHasError] = useState(false);
```

Comment:
I think you don`t need it here because you use it just for make error visible, so you can use the empty status of "errorMessage" to get the same result